src/js/components/Marathon.jsx

```diff
@@ -262,12 +213,12 @@ var Marathon = React.createClass({
               </a>
             </div>
             <NavTabsComponent
-              activeTabId={this.state.activeTabId}
+              activeTabId={state.activeTabId}
               className="navbar-nav nav-tabs-unbordered"
               tabs={tabs} />
             <ul className="nav navbar-nav navbar-right">
               <li>
-                <a href="#about">
+                <a href={"#" + router.getCurrentPathname() + "?modal=about"}>
```

Comment:
Can we use `<Link to={} />` here instead? See http://rackt.github.io/react-router/#Link .